martus-client/source/org/martus/client/swingui/dialogs/PureFxNotifyDlg.java

```diff
@@ -88,12 +91,22 @@ public void initialize(String title, String[] contents, String[] buttons, Map to
 		}
 	}
 
-	private TextArea createTextArea(String text)
+	private TextArea createTextArea(String content)
```

Comment:
This method is duplicated in 3 classes.   Since the classes dont share a common parent, maybe create a static method called from the 3 places.